UnityGLTF/Assets/UnityGLTF/Runtime/Scripts/GLTFSceneImporter.cs

```diff
@@ -381,9 +381,6 @@ public async Task LoadSceneAsync(int sceneIndex = -1, bool showSceneObj = true,
 
 				cancellationToken.ThrowIfCancellationRequested();
 
-				if (_gltfRoot.ExtensionsRequired != null && _gltfRoot.ExtensionsRequired.Contains("KHR_draco_mesh_compression"))
-					throw new NotSupportedException("Draco compression (using KHR_draco_mesh_compression) isn't currently supported. Use glTFast instead for loading meshes that use Draco compression.");
```

Comment:
Do we now log somewhere else if the Draco package is not installed instead? We should let people know what the problem is / how to deal with it / what to install from where

Reply:
in ImportMeshes line 119  > "Can't import model! Draco extension is needed. Please add com.atteneder.draco package to your project!""